interfaces/builtin/fwupd_test.go

```diff
@@ -123,16 +124,58 @@ func (s *FwupdInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	snippet, err = s.iface.PermanentSlotSnippet(s.slot, interfaces.SecurityDBus)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
-	snippet, err = s.iface.PermanentSlotSnippet(s.slot, interfaces.SecuritySecComp)
-	c.Assert(err, IsNil)
-	c.Assert(snippet, Not(IsNil))
 	snippet, err = s.iface.ConnectedSlotSnippet(s.plug, s.slot, interfaces.SecurityAppArmor)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
 	snippet, err = s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecurityAppArmor)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
-	snippet, err = s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecuritySecComp)
+}
+
+func (s *FwupdInterfaceSuite) TestPermanentSlotSnippetSecComp(c *C) {
+	slot := &interfaces.Slot{
+		SlotInfo: &snap.SlotInfo{
+			Snap:      &snap.Info{SuggestedName: "fwupd"},
+			Name:      "fwupd",
+			Interface: "fwupd",
+			Apps: map[string]*snap.AppInfo{
+				"app": {
+					Snap: &snap.Info{
+						SuggestedName: "fwupd",
+					},
+					Name: "app"}},
+		},
+	}
+
+	seccompSpec := &seccomp.Specification{}
+	err := seccompSpec.AddPermanentSlot(s.iface, slot)
 	c.Assert(err, IsNil)
-	c.Assert(snippet, Not(IsNil))
+	snippets := seccompSpec.Snippets()
+	c.Assert(len(snippets), Equals, 1)
+	c.Assert(len(snippets["snap.fwupd.app"]), Equals, 1)
+	c.Check(string(snippets["snap.fwupd.app"][0]), testutil.Contains, "bind\n")
```

Comment:
Hm we are looking for the same string here as we do in the test below. Might make sense to check for a comment as well so that we test that we actually have different "snippets" generated for fwupdPermanentSlotSecComp and fwupdConnectedPlugSecComp.